x-pack/test/api_integration/apis/asset_manager/tests/assets.ts

```diff
@@ -11,6 +11,7 @@ import { FtrProviderContext } from '../../../ftr_provider_context';
 import { createSampleAssets, deleteSampleAssets, viewSampleAssetDocs } from '../helpers';
 
 const ASSETS_ENDPOINT = '/api/asset-manager/assets';
+const DIFF_ENDPOINT = ASSETS_ENDPOINT + '/diff';
```

Comment:
nit: we can use a template string here
```suggestion
const DIFF_ENDPOINT = `${ASSETS_ENDPOINT}/diff`;
```